client/src/components/pages/LogIn/index.js

```diff
@@ -37,7 +37,7 @@ class LogIn extends Component {
             const {
               history: { push },
             } = this.props;
-            push(`/profile/parent/${result.data.id}`);
+            push(`/profile/parent/${result.data.message}`);
```

Comment:
this is not correct, the Route expects the parent's ID in the URL. If you need to pass the message to the component, here is how you do it:
- instead of doing `push` here, you do
```js
this.setState({ 
  redirectToParent: {
    id: result.data.id,
    msg: result.data.message
  }
});
```
- then in the render method, you do
```js
const { redirect } = this.state;
....
(this.state.redirect && (
    <Redirect to={{
        pathname: `/profile/parent/${redirect.id}`,
        state: { msg: redirect.msg}}
    />
))
```
- you'll have to make sure the `redirect` is removed from state. Use the componentWillUnmount
```js
componentWillUnmount() {
    this.setState({ redirect: null });
}
```
It's a bit more complicated and involves more typing, but this is the correct "react" way to do it (set state -> render -> pass props to child components) 🙂

Reply:
`message` is id from database I think this is an error from backend naming, we don't want to send message

Reply:
but thank you for this idea it's nice.

Reply:
oh I see 😀 well then you should fix it there :) it's the same repo so should be easy, can do it within this PR